### PR TITLE
Individual Reviews render, unstyled. Dates now format

### PR DESCRIPTION
**Pull request by philteves**

### CODE REVIEW

File: client/src/ratingsAndReviews/IndividualReview.jsx [hunk 1]
```diff
@@ -1,21 +1,30 @@
...
+//   review.rating: PropTypes.number,
+//   review.reviewer_name: PropTypes.string,
+//   review.date: PropTypes.string,
+//   review.summary: PropTypes.string,
+//   review.body: PropTypes.string,
+//   review.response: PropTypes.string,
+//   review.helpfulness: PropTypes.number
 };
 
 export default IndividualReview;
```
Comment (kalebal) [inline client/src/ratingsAndReviews/IndividualReview.jsx]: Needs a new line

**Review by kalebal — CHANGES_REQUESTED** [review]
Hate to do it, but IndividualReview needs a new line
Also, are you changing how you access props?

File: client/src/ratingsAndReviews/IndividualReview.jsx (hunk 1 above)
Comment (philteves) [reply client/src/ratingsAndReviews/IndividualReview.jsx]: Yes, latest commit has IndividualReview as stateful now & props refactored to reflect that. newline added too!

Comment (kalebal) [reply client/src/ratingsAndReviews/IndividualReview.jsx]: Sweet!

**Review by kalebal — APPROVED** [review]

**Pull request merged**